### DISCUSSION TITLE
Running avrdude remotely

### DISCUSSION THREAD

**Discussion by merlinblack** [avrdudes/avrdude#1252] [Ideas]
https://github.com/merlinblack/RemoteAVRDUDE

I made this in the last few days while on holiday (from coding for work :rofl: ) - mostly for fun and to play with c++17 features - however, it works well for what I use it for.

Basically a proxy for avrdude that understands some of the options enough to copy the needed files back and forth.

Perhaps other uses of avrdude will find it useful too.

**Comment by mcuee** [comment]
Thanks for the contribution. I tend to think this will not be included in the avrdude project.

**Comment by merlinblack** [comment]
"I tend to think this will not be included in the avrdude project." - Quite right - it's outside the scope of "do one thing and do it well". :smiley:

**Comment by dl8dtl** [comment]
I migrated it into a discussion item. I think that's more appropriate, and it prevent it from dropping down that much within the list of issues.

**Comment by stefanrueger** [comment]
Given the idea, as neat as it is, has not found traction closing the discussion as outdated.
